src/main/java/com/amazonaws/kinesisvideo/internal/producer/StreamEventMetadata.java

```diff
@@ -0,0 +1,60 @@
+package com.amazonaws.kinesisvideo.internal.producer;
+
+import javax.annotation.Nullable;
+
+/**
+ * Kinesis Video stream event metadata as provided by the native codebase.
+ */
+public class StreamEventMetadata {
+
+    // Current class version as described by the native struct.
+    public static final int STREAM_EVENT_METADATA_CURRENT_VERSION = 0;
+
+
+    // Version of the struct.
+    int mVersion;
+
+    // Optional s3 prefix.
+    @Nullable
+    String mImagePrefix;
+
+    // Optional optimization stating how many name/value pairs to be appended.
+    // NOTE: This is NOT optional if using mNames and mValues, else the arrays will not be used.
+    @Nullable
+    byte mNumberOfPairs;
+
+    // Optional custom data name/value pairs. String lengths must be <= MKV_MAX_TAG_NAME_LEN as defined in the native code.
+    @Nullable
+    String mNames[];
+    //
```

Comment:
Nit: Remove empty comment

Reply:
Not empty comment, this is to extend the "Optional custom data name/va..." comment to pertain to the lower lines too.